src/kakarot/accounts/library.cairo

```diff
@@ -493,6 +522,47 @@ namespace Internals {
         return ();
     }
 
+    // @notice Store the jumpdests of the contract.
+    // @param index The current free index in the jumpdests_ storage.
+    // @param jumpdests_len The length of the jumpdests.
+    // @param jumpdests The jumpdests of the contract.
+    func write_jumpdests{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}(
+        jumpdests_len: felt, jumpdests: felt*
+    ) {
+        alloc_locals;
+
+        if (jumpdests_len == 0) {
+            return ();
+        }
+
+        let (local base_address) = Account_valid_jumpdests.addr();
+        tempvar syscall_ptr = syscall_ptr;
+        tempvar jumpdests_len = jumpdests_len;
+        local pedersen_ptr: HashBuiltin* = pedersen_ptr;
+        local range_check_ptr = range_check_ptr;
```

Comment:
assign `local` before the `tempvar` as the locals are here to save the `.addr()` returned values (sort of cleanup of the previous line) while the `tempvar` are here to prepare the loop (next statement)